pandas/tests/indexing/multiindex/test_getitem.py

```diff
@@ -229,6 +229,26 @@ def test_frame_getitem_nan_multiindex(nulls_fixture):
     tm.assert_frame_equal(result, expected)
 
 
+def test_frame_getitem_nan_cols_multiindex(nulls_fixture):
+    # Slicing MultiIndex including levels with nan values, for more information
+    # see GH#25154
+    data = [[1, 2, 3], [4, 5, 6]]
+    index = ["First", nulls_fixture]
```

Comment:
Define the index only once if it is the same. You could remove the other definitions and write the data directly into the object.

Reply:
Thanks for the comments.
Suggested changes were implemented.